src/framework/ui/radio/radio.spec.tsx

```diff
@@ -13,78 +13,95 @@ import {
   StyleProviderProps,
 } from '@kitten/theme';
 import {
-  Radio,
-  Props,
+  Radio as RadioComponent,
+  Props as RadioProps,
 } from './radio.component';
 import * as config from './radio.spec.config';
+import { ReactTestInstance } from 'react-test-renderer';
 
-const StyledComponent = styled<Radio, Props>(Radio);
+const Radio = styled<RadioComponent, RadioProps>(RadioComponent);
 
-const Mock = (props?: Props): React.ReactElement<StyleProviderProps> => (
+const Mock = (props?: RadioProps): React.ReactElement<StyleProviderProps> => (
   <StyleProvider mapping={config.mapping} theme={config.theme} styles={{}}>
-    <StyledComponent {...props} />
+    <Radio {...props} />
   </StyleProvider>
 );
 
-const renderComponent = (props?: Props): RenderAPI => render(<Mock {...props}/>);
+const renderComponent = (props?: RadioProps): RenderAPI => {
+  return render(
+    <Mock {...props}/>,
+  );
+};
 
 describe('@radio: matches snapshot', () => {
 
   it('default', () => {
-    const component = renderComponent();
-    const { output } = shallow(component.getByType(Radio));
+    const component: RenderAPI = renderComponent();
```

Comment:
Should it be `ReactNode`?

Reply:
No. It's a type provided by react-native-testing-library ([typings](https://github.com/callstack/react-native-testing-library/blob/master/src/render.js)) for test components